packages/jspsych/src/modules/extensions.ts

```diff
@@ -1,31 +1,24 @@
-export interface JsPsychExtensionParameters {
-  [key: string]: any;
-}
-
-export interface JsPsychExtensionData {
-  [key: string]: any;
-}
-
-export interface JsPsychExtensionInfo {
+interface JsPsychExtensionInfo {
   name: string;
 }
 
 export interface JsPsychExtension {
+  info: JsPsychExtensionInfo;
```

Comment:
That demands a non-static info member. Intended?

Reply:
Oops. Not intended. I was having some git configuration issues and didn't realize this snuck into the commit.